packages/lib/preprocess/src/extract-queries/extract-queries.cjs

```diff
@@ -135,13 +138,15 @@ const strictBuild = process.env.VITE_BUILD_STRICT === 'true';
 const circularRefErrorMsg = 'Compiler error: circular reference';
 
 /**
- * @param {string} filename
+ * @param {string} content
  * @returns {Query[]}
  */
 const extractQueries = (content) => {
+	/** @type {Query[]} */
 	const queries = [];
 
-	queries.push(...extractExternalQueries(content));
+	// todo: second parameter is filename but we don't have that here?
+	queries.push(...extractExternalQueries(content, ''));
```

Comment:
You can make the second argument `filename` optional in the JSDoc comment on `extractExternalQueries`
https://stackoverflow.com/a/29441515/9080819

Reply:
I'm more wondering if we should be passing a file name into `extractQueries`. Not sure if we used to pass a file name in for the better error in `extractExternalQueries`

Reply:
Maybe, maybe not. But for now we should just maintain the argument being optional with the added types